### PR TITLE
replace stop_all_event_loops by 10 sec sleep period

### PR DESCRIPTION
**Pull request by kgaikwad**

### CODE REVIEW

**Comment by beav** [comment]
it looks like flake8 is failing:

```
yupana/processor/processor_utils.py:21:1: I202 Additional newline in a group of imports. 'from time import sleep' is identified as Stdlib and 'import logging' is identified as Stdlib.
```

If you run `flake8  yupana/processor/processor_utils.py ` it should test the file locally I think and you can correct the issue.

**Comment by kgaikwad** [comment]
> If you run `flake8 yupana/processor/processor_utils.py ` it should test the file locally I think and you can correct the issue.

It will sound weird but for me it is running locally just fine without any failure (python3-flake8-3.7.7). Debugging.

**Comment by beav** [comment]
This worked for me when I tested it.

**Review by beav — APPROVED** [review]

**Comment by beav** [comment]
@kgaikwad I'm not sure why the code coverage test failure is happening. I think that is the last part needed to get this merged.

File: yupana/processor/tests_report_consumer.py [hunk 1]
```diff
@@ -139,3 +140,25 @@ def test_save_and_ack_success(self):
         coro = asyncio.coroutine(self.save_and_ack)
         event_loop.run_until_complete(coro())
         event_loop.close()
+
+    async def async_test_listen_for_messages(self):
+        """Test listen for messages via kafka."""
```
Comment (kgaikwad) [inline yupana/processor/tests_report_consumer.py:145]: @beav, I have added a test for method "listen_for_message" could you please confirm whether it makes sense or not?

Comment (beav) [reply yupana/processor/tests_report_consumer.py:145]: looks ok to me and tests pass

**Comment by beav** [comment]
@kgaikwad the change looks good to me. you should have access to merge this and override the check.

**Comment by kgaikwad** [comment]
> @kgaikwad the change looks good to me. you should have access to merge this and override the check.

Thank you @beav for review. Merging this pull-request.

**Pull request merged**